webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/NetworkingInterfaces/NetworkingInterfaces.scss

```diff
@@ -0,0 +1,15 @@
+.interface-accordion {
+  .interface-name svg {
+    margin-left: 5px;
+  }
+  .pf-l-grid__item {
+    text-align: left;
+    white-space: pre-wrap;
+  }
+  .pf-c-accordion__toggle-text {
+    width: 100%;
+  }
+}
+.nic-empty-state{
+  padding-left: 16px;
+}
```

Comment:
Small nit: we generally have a line ending at the end of the file. The reason is that if you add new lines at the end then you're not changing the last line.

Ideally we'd have some automated check for CI to fail on this.

Reply:
prettier/eslint does enforce this, just not for `scss` files.

Reply:
Thanks for that clarification. I did have some vague recollection that we had it and was surprised this slipped through.

Reply:
Fixed